src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/NullToVisibilityConverter.cs

```diff
@@ -15,7 +15,14 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             if (targetType == typeof(Visibility))
             {
-                return value == null ? Visibility.Collapsed : Visibility.Visible;
+                if (CollapseEmptyString && value.GetType() == typeof(string))
```

Comment:
Something seems wrong.

This class is `NullToVisibilityConverter` which strongly implies that it converts `null` values to some `Visibility` value and that `value` will be `null`.

If so, `value.GetType()` will generate a `NullReferenceException`.  Sure, you could first check if `value` is `null` like the original code did, but then this class seems to be the wrong place for converting a non-`null` value to a `Visibility` value.

What's wrong?

Reply:
yes, good point...my new code path doesn't work for null values. It only works for empty strings. 
it should support both.
In our usage, it appears that Tags is always empty, not null...so i didn't catch that with runtime testing.